python/python/tests/test_async_client.py

```diff
@@ -5559,6 +5559,49 @@ async def test_getex(self, redis_client: TRedisClient):
         )
         assert await redis_client.ttl(key1) == -1
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lolwut(self, redis_client: TRedisClient):
+        result = await redis_client.lolwut()
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(parameters=[50, 20])
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(6)
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(5, [30, 4, 4])
+        assert "Redis ver. " in result
+
```

Comment:
Can we add a test for if an empty parameter list is passed?
```suggestion
        # passing an empty parameters list has no effect
        result = await redis_client.lolwut(parameters=[])
        assert "Redis ver. " in result
```